src/fasttext.cc

```diff
@@ -69,6 +69,10 @@ void FastText::getWordVector(Vector& vec, const std::string& word) const {
   if (ngrams.size() > 0) {
     vec.mul(1.0 / ngrams.size());
   }
+
+  if(normalise) {
+    vec.normalise();
```

Comment:
Could you please use
```
real norm = vec.norm();
if (norm > 0.0) {
  vec.mul(1.0 / norm);
}
```
instead? And remove the changes in `vector.cc` and `vector.h`. Thanks.